frontend/client/views/secrets/index.vue

```diff
@@ -120,9 +120,9 @@
                   </td>
 
                   <td width="68">
-                    <a v-if="editMode && currentPathType === 'Secret'" @click="deleteItem(index)">
```

Comment:
The key/value deletion and secret deletion should not interact with each other in code.

Let's just make this two buttons. Two <a> elements.

`v-if="currentPathType === 'Secret' && editMode"` for the circle icon
`v-if="currentPathType === 'Path' && entry.type === 'Secret'` for the second icon.

They will be mutually exclusive.